models/contacts.js

```diff
@@ -1,19 +1,66 @@
-// const fs = require('fs/promises')
+const fs = require('fs/promises')
+const path = require('path')
+const Joi = require('joi')
 
-const listContacts = async () => {}
+const contactsPath = path.join(__dirname, '..', 'db', 'contacts.json')
 
-const getContactById = async (contactId) => {}
+const listContacts = async () => {
+    const data = await fs.readFile(contactsPath, 'utf-8')
+    return JSON.parse(data)
+}
+
+const getContactById = async (contactId) => {
+    const contacts = await listContacts()
+    const contact = contacts.find((c) => c.id === contactId)
```

Comment:
please dont use `c` as variable name. Use something with meaning. In your case `c` is `contact`. It's easier to understand the code with proper naming ;)